ansible_collections/arista/avd/examples/single-dc-multipod-l3ls/README.md

```diff
@@ -0,0 +1,287 @@
+---
+# This title is used for search results
+title: AVD example for a single data center using multiple pods for l3ls
+---
+<!--
+  ~ Copyright (c) 2023-2024 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+
+# AVD example for a single data center with multipod using L3LS
+
+## Introduction
```

Comment:
Please add a small section for how to install this example (I know it is the same as for the rest).